This patch adds the configuration needed to used the EDBG debugger of an
XplainedPro in PDI mode, e.g. such as the one on the XMEGA A1U Xplained Pro
--- a/src/avrdude.conf.in
+++ b/src/avrdude.conf.in
@@ -1095,6 +1095,14 @@
 ;
 
 programmer
+  id    = "xplainedpro_pdi";
+  desc  = "Atmel AVR XplainedPro in PDI mode";
+  type  = "jtagice3_pdi";
+  connection_type = usb;
+  usbpid = 0x2111;
+;
+
+programmer
   id    = "xplainedmini";
   desc  = "Atmel AVR XplainedMini in ISP mode";
   type  = "jtagice3_isp";
